src/store/store.js

```diff
@@ -0,0 +1,49 @@
+import Vue from 'vue';
+import Vuex from 'vuex';
+
+Vue.use(Vuex);
+
+export const store = new Vuex.Store({
+    state: {
+        cities: [ 
+            {id: 0, name: 'Satu Mare'},
```

Comment:
In the future task, we'll introduce the linting, so everytime there is a linting issue, it will cause a warning :) We'll use the airbnb configuration, which requires you to add an extra space before and after the '{'